### PR TITLE
feature/archive account

### PR DESCRIPTION
**Pull request by sywhb**
- **allow omnivore_admin to delete filters**
- **add archived status to user account**
- **redirect archived user to /export page when login**

### CODE REVIEW

File: packages/api/src/jobs/rss/refreshAllFeeds.ts [hunk 1]
```diff
@@ -40,12 +40,11 @@ export const refreshAllFeeds = async (db: DataSource): Promise<boolean> => {
       FROM
         omnivore.subscriptions s
       INNER JOIN
-        omnivore.user u ON u.id = s.user_id
+        omnivore.user u ON u.id = s.user_id AND u.status = $4
```
Comment (sywhb) [inline packages/api/src/jobs/rss/refreshAllFeeds.ts:43]: Optimized the query a little bit here

File: packages/api/src/services/library_item.ts [hunk 2]
```diff
@@ -1039,7 +1039,10 @@ export const updateLibraryItemReadingProgress = async (
   }
 
   const updatedItem = result[0][0]
-  await pubsub.entityUpdated<ItemEvent>(EntityType.ITEM, updatedItem, userId)
+  if (updatedItem.readingProgressBottomPercent === 100) {
```
Comment (sywhb) [inline packages/api/src/services/library_item.ts:1042]: Here I try to reduce the number of PAGE_UPDATE event being sent so we only send the event when the item is fully read or reset to unread

File: packages/api/src/routers/auth/google_auth.ts [hunk 3]
```diff
@@ -158,15 +157,16 @@ export async function handleGoogleWebAuth(
       }
     }
 
+    let redirectURL = `${baseURL()}${
+      user.status === StatusType.Archived ? '/export' : DEFAULT_HOME_PATH
```
Comment (jacksonh) [inline packages/api/src/routers/auth/google_auth.ts:161]: Can we change this to `/account-archived`.

**Comment by jacksonh** [comment]
Just one small change on the path

**Review by jacksonh — APPROVED** [review]

**Pull request merged**